frontend/src/components/courses/evaluations/Evaluate.tsx

```diff
@@ -0,0 +1,257 @@
+import React, { useContext, useState } from "react";
+import { PageTitle } from "../../utils/PageTitle";
+import { api, useBetterQuery } from "../../../services/apiService";
+import { Redirect } from "react-router-dom";
+import { Form, Button, Card, Container, Row, Col } from "react-bootstrap";
+import { Question } from "../../../models/courses/question"
+import { useField, Formik, FormikProps } from "formik";
+import { ToastContext, ToastLevel } from "../../utils/Toast";
+
+export const EvaluateCourse = ({ course }) => {
+    const { data: questions, error, status } = useBetterQuery<Question[]>(
+        ["courses.questions", course.form],
+        api.courses.forms.questions.list,
+    );
+
+    if (status === "loading") return <p>Chargement des cours</p>;
+    else if (status === "error") {
+        return `Something went wrong: ${error}`;
+    } else if (status === "success" && questions) {
+        return (
+            <Container>
+                <PageTitle>Cours</PageTitle>
+                <QuestionsForm questions={questions} course={course} />
+            </Container>
+        );
+    }
+
+    return null;
+};
+
+interface Values {
```

Comment:
`Values` seems a bit too generic

Reply:
Replaced it with EvalSubmission which seems more relevant